roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-cnv-operator.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Create the CNV Operator namespace
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: openshift-cnv
+
+- name: Create CNV OperatorGroup kubevirt-hyperconverged-group
+  kubernetes.core.k8s:
+    state: present
+
+    definition:
+      apiVersion: operators.coreos.com/v1
+      kind: OperatorGroup
+      metadata:
+        name: kubevirt-hyperconverged-group
+        namespace: openshift-cnv
+      spec:
+        targetNamespaces:
+          - openshift-cnv
+
+- name: Create CNV Subscription
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: hco-operatorhub
+        namespace: openshift-cnv
+      spec:
+        source: redhat-operators
+        sourceNamespace: openshift-marketplace
+        name: kubevirt-hyperconverged
+
+- name: Wait
+  ansible.builtin.pause:
+    seconds: 60
+
+- name: Get Install Plan Name
+  retries: 10
+  delay: 10
+  ansible.builtin.command: oc get subscriptions/hco-operatorhub --namespace openshift-cnv --output=jsonpath='{$.status.installplan.name}'
```

Comment:
Use `kubernetes.core.k8s_info` instead of `oc` to get the installplan name.

Reply:
Noted, will look into that later.